tests/copybw.cpp

```diff
@@ -144,8 +144,7 @@ int main(int argc, char *argv[])
     ASSERT_NEQ(init_buf, (void*)0);
     init_hbuf_walking_bit(init_buf, size);
 
-    gdr_t g = gdr_open();
-    ASSERT_NEQ(g, (void*)0);
```

Comment:
it looks like you removed the ASSERT_NEQ(). It's not a big deal gdr_open_safe() already handles the error, but it breaks the pattern without a reason.
I suggest to either remove ASSERT_NEQ() from all gdr_open_safe() call sites, or to preserve the one here.

Reply:
Since ASSERT_NEQ loss its meaning after we have gdr_open_safe, I choose to remove it from all call sites.